### ISSUE TITLE
Not evaluation with backbone DenseNet

### ISSUE THREAD

**Issue by mariaculman18** [fizyr/keras-retinanet#1103]
I trained a model using as a backbone the DenseNet-121. I am trying to evaluate the model's mean average precision (mAP) with the training dataset but I am getting `mAP: 0.0000`. 

Versions:
RetinaNet 0.5.1
Keras 2.2.4
TensorFlow GPU 1.12

While training the model I got the training loss, the validation loss and the validation mAP by using `--compute-val-loss` and `--val-annotations dataset/test.csv`. I converted the trained model to an inference model with `retinanet-convert-model --backbone=densenet121 [path to trained model (*.h5)] [path to save inference model (*.h5)]`. Now I want to obtain the mAP with the training dataset, so I am doing `retinanet-evaluate csv dataset/train.csv dataset/classes.csv [path to inference model (*.h5)] --backbone=densenet121`. But I am getting `mAP: 0.0000`. 

Any idea what can I do?

**Comment by hgaiser** [comment]
Hey, as you have already seen, this issue is discussed in https://github.com/fizyr/keras-retinanet/issues/647 . Let's keep the discussion there.
